### PR TITLE
Remove unnecessary files

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/2329

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -238,16 +228,9 @@ stages:
...
-          # Reenable tagging when stable package versions can be generated.
-          # https://github.com/dotnet/arcade-services/issues/59
-          # - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'production')) }}:
-            # - powershell: eng\create-tag.ps1 -AccessToken $(System.AccessToken)
-
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
-      # Symbol validation isn't being very reliable lately. This should be enabled back
-      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
```
Comment (premun) [inline azure-pipelines.yml:250]: This is removed but the symbol validation below stays turned off. Should we not keep it or flip the boolean?

**Review by premun — DISMISSED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (dkurepa) [reply azure-pipelines.yml:250]: We can try flipping it on, I'll run these changes on the internal pipeline, see if they behave as expected

Comment (dkurepa) [reply azure-pipelines.yml:250]: https://dev.azure.com/dnceng/internal/_build/results?buildId=2194790&view=results

Comment (dkurepa) [reply azure-pipelines.yml:250]: Actually, I'll add the missing tests in a separate PR, think that will be better

**Review by premun — APPROVED** [review]

**Pull request merged**